dist/scss/wds-components/global-footer/_breakpoint-l-down.scss

```diff
@@ -0,0 +1,70 @@
+@import 'variables';
+
+.wds-global-footer {
+	@media #{$wds-breakpoint-l-down} {
```

Comment:
Breakpoints in this PR are outdated.

Reply:
Yeah, but it's a nice opportunity to replace them:
> NOTICE: The goal is to use those breakpoints in all our products. Please replace the old breakpoints with the new ones whenever and wherever possible.
from http://fandomdesignsystem.com/#/base-styles/wds-breakpoints